renderdoc/driver/shaders/dxbc/dxbc_bytecode.cpp

```diff
@@ -380,16 +380,36 @@ DXBC::Reflection *Program::GuessReflection()
         cb.descriptor.flags = 0;
         cb.descriptor.numVars = numVecs;
 
-        cb.variables.reserve(numVecs);
+        bool isArray = desc.bindCount > 1;
+        if(isArray)
+        {
+          // If the constant buffer is an array, then we need to add an entry for the struct
+          // itself. This mimics what is loaded for a constant buffer array when reflection
+          // information is not stripped.
+          DXBC::CBufferVariable var;
+          var.name = cb.name;
+          var.descriptor.name = cb.name;
+          var.descriptor.offset = 0;
+          var.descriptor.flags = 2;
+          var.type.descriptor.varClass = DXBC::VariableClass::CLASS_STRUCT;
+          var.type.descriptor.type = DXBC::VariableType::VARTYPE_VOID;
+          var.type.descriptor.rows = 1;
+          var.type.descriptor.cols = 4;
+          var.type.descriptor.elements = 1;
+          var.type.descriptor.members = numVecs;
+          var.type.descriptor.bytesize = 4 * sizeof(float);
```

Comment:
Should there be an extra ` * numVecs` on this `bytesize` calculation?

Reply:
The other way around - `members` should be 1